src/subdomains/core/aml/enums/aml-error.enum.ts

```diff
@@ -62,6 +62,7 @@ export enum AmlError {
   BANK_RELEASE_DATE_MISSING = 'BankReleaseDateMissing',
   IP_COUNTRY_MISMATCH = 'IpCountryMismatch',
   TRADE_APPROVAL_DATE_MISSING = 'TradeApprovalDateMissing',
+  INTERMEDIARY_WITHOUT_SENDER = 'IntermediaryWithoutSender',
```

Comment:
The error should be renamed to `CustomerNameMissing` or something like that. This would be better for the amlError, the amlReason is perfectly fine with `IntermediaryWithoutSender` and should not be changed